tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-torch1110-cuda102.yml

```diff
@@ -0,0 +1,15 @@
+trigger: none

```

Comment:
Why are we adding this file? How do you feel about leaving the orttrainer tests (old tests being invoked from build.py) depend on torch 1.9 (as they have been so far). We can only change the torch version for ortmodule tests?